pkg/engine/engine.go

```diff
@@ -158,6 +161,23 @@ func (e *Engine) alterFuncMap(t *template.Template) template.FuncMap {
 		return val, nil
 	}
 
+	// Add the 'tpl' function here
+	funcMap["tpl"] = func(tpl string, vals chartutil.Values) (string, error) {
+		r := renderable{
+			tpl:  tpl,
+			vals: vals,
+		}
+
+		templates := map[string]renderable{}
+		templates["template"] = r
+
+		result, err := e.render(templates)
```

Comment:
I wonder if doing a `t.Parse()` here instead might cause it to inherit scope. I'm not sure how best to work around the loss of context...